docs/runtimes/application.ipynb

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
<p>set_internal_app_port was change to set_internal_application_port</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>